integration-tests/src/test/java/tachyon/master/lineage/LineageMasterIntegrationTest.java

```diff
@@ -142,7 +142,7 @@ public void lineageCompleteAndAsyncPersistTest() throws Exception {
           fileInfo.getPersistenceState());
 
       // sleep and wait for worker to persist the file
-      CommonUtils.sleepMs(5);
+      CommonUtils.sleepMs(200);
```

Comment:
Is there a reason the thread that does the persistence is not an executor thread? (That would by default execute the heartbeat method that does the persistence in a blocking fashion every X seconds).